assembly/pom.xml

```diff
@@ -22,12 +22,11 @@
   <parent>
     <groupId>org.apache.shindig</groupId>
```

Comment:
groupId should be prefixed with org.wso2.

Reply:
Done